tests/test_order.py

```diff
@@ -215,17 +232,61 @@ def test_track_status_fail(order_mock, status, requests_mock):
 
 
 def test_estimate_order(order_parameters, auth_mock, requests_mock):
-    url_order_estimation = (
-        f"{auth_mock._endpoint()}/workspaces/{auth_mock.workspace_id}/orders/estimate"
-    )
-    requests_mock.post(url=url_order_estimation, json={"data": {"credits": 100}})
+    url_order_estimation = f"{auth_mock._endpoint()}/v2/orders/estimate"
+    expected_payload = {
+        "summary": {"totalCredits": 38, "totalSize": 0.1, "unit": "SQ_KM"},
+        "results": [{"index": 0, "credits": 38, "unit": "SQ_KM", "size": 0.1}],
+        "errors": [],
+    }
+    expected_output = {
```

Comment:
nitpick: it feels like this exact dict is repeated 2-3 times, maybe makes sense to make it into a variable?

Reply:
Good question. The idea is to make tests as independent as possible. If we want in the future to change tests I would not like to need to affect the other ones. In this case, Do you think that sharing variables is recommended bad among tests?

Reply:
Hi yes in general, tests should be independent of each other. But in this case, this 4 occurrences are in the forms of the same function "estimate_order" tests, i think the `expected_output` variable value wont change (if this expected response change in 1, and it doesnt change in the other 3, i think there is something wrong)
But it's just a suggestion.